### DISCUSSION TITLE
Has ALPACA algorithm changed in new versions of Slicer?

### DISCUSSION THREAD

**Discussion by Der-pina** [SlicerMorph/SlicerMorph#375] [Q&A]
Hi :) 
I have a small problem with Alpaca which I can't figure out. I've noticed that the parameters that performed well for the version of Slicer 5.0.2 do not work for the version 5.6.2. I see that in the 5.6.2 version there are some additional parameters that can be adjusted, such as Poisson point subsampling, FPFH neighbour count, but I don't fully understand what they do.  I'd appreciate any tip :)

**Comment by muratmaga** [comment]
Yes,  we now use  an ITK based implementation of ransac and fpfh libraries, instead of the open3d that was used originally. 
Aa such,  the defaults are little different.  However the results should be even better now.
